index.html

```diff
@@ -15,5 +15,89 @@
 
 <body>
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<div class='container'>
+
+  
+  
+  <nav class="navigation">
+    <div class='aNav'>
+      <a class='a1' href='services.html'>Service</a>
+      <a href=''>Product</a>
+      <a href=''>Vision</a>
+      <a href=''>Features</a>
+      <a href=''>About</a>
+      <a href=''>Contact</a>
+    </div>
+    <div class='logoDiv'>
+      <img src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+  
+  </nav>
+
+  <header class ='innovationDiv'>
+      <div class='blockInn'>
+        <h1 class='innovation'>Innovation
+        On
+        Demand
+        </h1>
+      <button class='getStarted'>Get Started</button>
+      </div>
+      <div>
+      <img class=codePic src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+  </header>
+
+  <hr>
+
+  <section class="top-content">
+    <div class='featuresDiv'>
+      <h2 class='features'>Features</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in
+        interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque
+        quis.</p>
+    </div>
+    <div class='aboutDiv'>
+      <h2 class='about'>About</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in
+        interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque
+        quis.</p>
+    </div>
+  </section>
+
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+  <section class="bottom-content">
+    <div class='services'>
+      <h2>Services</h2>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+    Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+
+    <div class='product'>
+      <h2>Product</h2>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+    Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+    <div class='vision'>
+      <h2>Vision</h2>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+    Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+  </section>
+  
+  <hr>
+  
+  <section class='contact'>
+    <h2>Contact</h2>
+    123 Way 456 Street
+    <p>Somewhere, USA</p>
+    <p class='web'>1 (888) 888-8888</p>
```

Comment:
```suggestion
    <a href="tel:18888888888">1 (888) 888-8888</a>
```
Putting this in an anchor tag will allow the user to click on it and be directed to the calling app on their phone. It is more semantic and lets the browser know that this is a phone number. You can do something similar with the email address below this line and put it in an anchor tag as well